internal/pkg/plugin/githubactions/python/validate.go

```diff
@@ -0,0 +1,53 @@
+package python
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+// validate validates the options provided by the core.
+func validate(opts *Options) []error {
+	retErrors := make([]error, 0)
+
+	// owner/repo/branch
+	if opts.Owner == "" {
+		retErrors = append(retErrors, fmt.Errorf("owner is empty"))
+	}
+	if opts.Repo == "" {
+		retErrors = append(retErrors, fmt.Errorf("repo is empty"))
+	}
+	if opts.Branch == "" {
+		retErrors = append(retErrors, fmt.Errorf("branch is empty"))
+	}
+
+	// language
+	if opts.Language == nil {
+		retErrors = append(retErrors, fmt.Errorf("language is empty"))
+	}
+	if errs := opts.Language.Validate(); len(errs) != 0 {
+		for _, e := range errs {
+			retErrors = append(retErrors, fmt.Errorf("language is invalid: %s", e))
+		}
+	}
+
+	return retErrors
+}
+func parseAndValidateOptions(options map[string]interface{}) (*Options, error) {
```

Comment:
https://github.com/merico-dev/stream/pull/359#discussion_r835904924

@summingyu Please open a new pr to fix the issue mentioned at the conversation above.